apiserver/plane/app/views/asset/v2.py

```
@@ -718,3 +718,93 @@ def get(self, request, slug, asset_id):
             id=asset_id, workspace__slug=slug, deleted_at__isnull=True
         ).exists()
         return Response({"exists": asset}, status=status.HTTP_200_OK)
+
+
+class DuplicateAssetEndpoint(BaseAPIView):
+    def get_entity_id_field(self, entity_type, entity_id):
```

Pylint (3.3.7)

[refactor] Too many return statements (7/6)

(R0911)